prime-router/docs/api/components/security_schemes.yml

```diff
@@ -0,0 +1,14 @@
+OktaAuth:
+  type: oauth2
+  flows:
+    authorizationCode:
+      authorizationUrl: https://hhs-prime.okta.com/oauth/authorize
+      tokenUrl: https://hhs-prime.okta.com/oauth/token
+      scopes:
+        ANONYMOUS: Grants read access
+        FUNCTION: Grants write access to single org
+        ADMIN: Grants access to admin operations
+Server2Server: # aka two-legged aka fhirauth
```

Comment:
The older specs use:
```
    ApiKeyAuth:
      type: apiKey
      in: header
      name: x-functions-key
```
Is this still valid? Correct? Should we add any of these descriptors?

Reply:
I will add ApiKeyAuth into the doc. Server2Server has some overlap with it, but it has not been merged in yet. I'm including it since it's used by the new endpoints.